src/mesh/Mesh.hpp

```diff
@@ -191,6 +191,13 @@ class Mesh {
   /// Computes the boundingBox for the vertices.
   void computeBoundingBox();
 
+  /// Computes if a quad is convex and returns and ordered set of vertices.
+  void computeQuadConvexityFromPoints(std::array<int,4> &hull) const;
+
+  /// Computes if a quad is convex and returns and ordered set of vertices.
+  //void computeQuadEdgeOrder(Vertex &verticesOne, Vertex &verticesTwo, Vertex &verticesThree, Vertex &verticesFour) const;
+  void computeQuadEdgeOrder(std::array<int,4> &edgeList, std::array<int,4>  &vertexList) const;
```

Comment:
How is invalid input treated in this function? Meaning duplicated edges etc.

Reply:
These are now checked in `SolverInterfaceImpl` to ensure they are unique before calling `computeQuadEdgeOrder`.